numcodecs/zarr3.py

```diff
@@ -184,30 +195,18 @@ async def _encode_single(self, chunk_ndbuffer: NDBuffer, chunk_spec: ArraySpec)
 
 
 def _add_docstring(cls: type[T], ref_class_name: str) -> type[T]:
```

Comment:
All this can be deleted once the changes in this PR are applied to all the other codecs